repo-projects/runtime.proj

```diff
@@ -17,6 +17,9 @@
 
     <!-- May stabilize. -->
     <AllowDotNetFinalVersionKindOverride>true</AllowDotNetFinalVersionKindOverride>
+    
+    <!-- The bootstrapping build drops its log in the root artifacts/log folder for the repo build. -->
+    <MoveNoConfigLogs>true</MoveNoConfigLogs>
```

Comment:
Why is that? Can't the bootstrap build put them under `artifacts\log\$(Configuration)\bootstrap\`?